src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -80,6 +80,17 @@ public static TestCommand FromArgs(string[] args, string msbuildPath = null)
                 msbuildPath);
         }
 
+        private static string DefaultVerbosity(AppliedOption parsedTest)
```

Comment:
extreme nit: rename from `parsedTest` to something like `options`?

Reply:
renamed